sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -1,13 +1,28 @@
-// TODO: When all icon are replace to font awesome,
-// replace  icon?: string | string[] with
-// icon?:string[];
+import { RouteLocationRaw } from "vue-router";
+
+/**
+ * ! @deprecated Use MenuItemModel and once MenuModel removed from all delete the MenuModel.
```

Comment:
Please change the comment as shown below to cause the proper effect in the MenuModel. VSCode will add a "dash" to the deprecated class.
![image](https://user-images.githubusercontent.com/61259237/190521517-95be2404-495f-40ce-a092-6cdf42519ea1.png)

Reply:
done